content/200-concepts/100-components/01-prisma-schema/06-relations/420-relation-mode.mdx

```diff
@@ -210,6 +210,28 @@ When you introspect a database with the `db pull` command with the `prisma` rela
 
 When you apply changes to your Prisma schema with Prisma Migrate or `db push` with the `prisma` relation mode enabled, Prisma will not use foreign keys in your database.
 
+### Indexes
+
+In relational databases that use foreign key constraints, the database usually also implicitly creates an index for the foreign key columns. For example, [MySQL will create an index on all foreign key columns](https://dev.mysql.com/doc/refman/8.0/en/create-table-foreign-keys.html). This is to allow foreign key checks to run fast and not require a table scan.
```

Comment:
This link highlights exactly the correct sentence:
```suggestion
In relational databases that use foreign key constraints, the database usually also implicitly creates an index for the foreign key columns. For example, [MySQL will create an index on all foreign key columns](https://dev.mysql.com/doc/refman/8.0/en/constraint-foreign-key.html#:~:text=MySQL%20requires%20that%20foreign%20key%20columns%20be%20indexed%3B%20if%20you%20create%20a%20table%20with%20a%20foreign%20key%20constraint%20but%20no%20index%20on%20a%20given%20column%2C%20an%20index%20is%20created.). This is to allow foreign key checks to run fast and not require a table scan.
```

Reply:
I'm a bit wary of linking to specific highlights, it seems brittle if the text changes slightly.

The link itself was wrong though, so I've updated it

Reply:
Hm, thinking about it, it will fall back to the section heading anyway, so it's probably fine.